src/libcore/iter/traits.rs

```diff
@@ -706,7 +706,7 @@ pub trait ExactSizeIterator: Iterator {
     /// ```
     /// #![feature(exact_size_is_empty)]
     ///
-    /// let mut one_element = 0..1;
+    /// let mut one_element = std::iter::once(0);
```

Comment:
Why is this related to this PR?

Reply:
Because using `0..1` means the code wasn't actually using `ExactSizeIterator`; the inherent method was getting picked up instead.  For normal code that's not a problem--they do exactly the same thing--but I figured the doctest should actually use the method it's documenting.